test/Riok.Mapperly.Tests/TestHelper.cs

```diff
@@ -80,4 +71,27 @@ private static CSharpCompilation BuildCompilation(NullableContextOptions nullabl
 
         return CSharpCompilation.Create("Tests", syntaxTrees, references, compilationOptions);
     }
+
+    public static GeneratorDriver TrackingGeneratorDriver()
```

Comment:
Instead of this method, I think `GenerateTracked(Compilation)` could be exposed, which calls `RunGenerators` on the driver (each caller of this method calls `RunGenerators` anyway).